src/utils/index.js

```diff
@@ -59,8 +59,8 @@ function isDefined(value) {
   return value !== undefined && value !== null
 }
 
-export function allFilesAccepted({ files, accept, minSize, maxSize, multiple }) {
-  if (!multiple && files.length > 1) {
+export function allFilesAccepted({ files, accept, minSize, maxSize, multiple, maxFiles }) {
```

Comment:
Let's add some tests for this fn as well.

Reply:
The test has been added.